### ISSUE TITLE
Scalings are not reflecting changes with dynamic preview on + they are creating a duplicate profiles after saving changes.

### ISSUE THREAD

**Issue by ShiftyKiwi** [XIV-Tools/CustomizePlus#122]
After the recent updates, there are several issues that have persisted with editing scalings assigned to different characters. 

The first issue that I had noticed, was that when going into the editing menu within a scaling profile that I had saved, the changes were not being reflected with the automatic/dynamic preview option checked. Strangely, even though the changes were not reflected, the x, y, z values that I had edited, would stay the same. For example: changing x, y, z, on my root would stay to the new values that I had increased them to from 2, 2, 2, but their values in actual size, would still be the original 1, 1, 1.

Additionally, after trying to save the edits that I had made in the scaling profile, this would cause an additional, identical scaling profile, to pop up into the list of other scaling profiles.

**Comment by moonlitglare** [comment]
I am having this issue as well.

**Comment by johsan09** [comment]
I'm having that problemm too

**Comment by StoiaCode** [comment]
Is this still happening? I couldnt reproduce it for myself for some time now.

**Comment by ShiftyKiwi** [comment]
No, whatever updates were implemented over these last few months seems to have fixed this particular bug. Thanks for y'all's hard work!
